birdy/native/client.py

```diff
@@ -0,0 +1,217 @@
+import types
+from collections import OrderedDict
+from copy import copy
+
+import six
+from boltons.funcutils import FunctionBuilder
+from owslib.util import ServiceException
+from owslib.wps import WPS_DEFAULT_VERSION, WebProcessingService, SYNC
+
+from birdy.exceptions import UnauthorizedException
+from birdy.native.utils import build_doc, convert_input_value, convert_output_value, delist
+from birdy.native.converters import default_converters
+
+
+# TODO: Add credentials and tokens
```

Comment:
Update todos. Do they still make sense ? Maybe the log is unnecessary with the improved error handling.